### DISCUSSION TITLE
Filter di Halaman Tantangan

### DISCUSSION THREAD

**Discussion by sooluh** [upkoding/upkoding#28] [Ideas]
Aku kesusahan buat ngeliat tantangan yg udah aku kerjain atau belum dikerjakan sama sekali di halaman semua tantangan, mungkin menambahkan kolom filter untuk melihat tantangan yang sama sekali belum dikerjakan, sedang proses pengerjaan maupun sudah selesai dikerjakan bisa membantu mempermudah itu hehe 😅

Sebelumnya, terimakasih 🙏🏻

**Comment by ekaputra07** [comment]
Betul nih @sooluh, saya sudah tambahkan tanda mana proyek/challenge yg udh dikerjakan mana yg belum. Tp sepertinnya kalau bisa difilter lebih praktis ya. Mudah2an bisa tersedia di next release 🙏
